RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -36,13 +38,36 @@ namespace {
     
   inline float chi2n(reco::Track const & tk) { return tk.normalizedChi2();}
 
+  inline float relPtErr(reco::Track const & tk) {
+    return  float(tk.ptError())/float(tk.pt());
```

Comment:
you are right !
I forgot it :(
I think I should update this PR (sorry) and the https://github.com/cms-sw/cmssw/pull/14996 as well